src/testdir/test_modeline.vim

```diff
@@ -258,6 +258,7 @@ func Test_modeline_fails_always()
   call s:modeline_fails('t_ce', 't_ce=Something()', 'E520:')
   call s:modeline_fails('t_sr', 't_sr=Something()', 'E520:')
   call s:modeline_fails('t_8b', 't_8b=Something()', 'E520:')
+  call s:modeline_fails('findexpr', 'findexpr=Something()', 'E520:')
```

Comment:
This should come before the `" only check a few terminal options` comment, and probably between `formatprg` and `exrc`